scripts/spine_generic_test_split_for_csa_drift_monitoring.yaml

```diff
@@ -0,0 +1,68 @@
+# This file contains the frozen test split for the spine-generic `data-multi-subject` dataset on which the 
+# morphometric variability (CSA) was computed on using while developing the contrast-agnostic model. 
+# To monitor the drift in the CSA variability as the model evolves, we freeze the test split and use it to
+# evaluate the model at different stages of its evolution (i.e. when new datasets and pathologies are added).
+# 
+# How this split was obtained:
+# ```python
+# from sklearn.model_selection import train_test_split
+# import numpy as np
+#
+# np.random.seed(seed=50)
+# test_ratio = 0.2
+# train_subjects, test_subjects = train_test_split(all_subjects, test_size=test_ratio)
+# ```
+# 
+# NOTE: `all_subjects` is a list of all the subjects in the spine-generic dataset
+# https://github.com/spine-generic/data-multi-subject
```

Comment:
maybe a specific release or tag could be added as in the future new subjects can be added, and `train_test_split` would thus result in a different test split

Reply:
good point! I don't see a release associated with the current master, do you know if there's a way to reference the current state of the dataset? Just referring to the final commit, maybe that could work?

Reply:
There've been several recent merges to master (such as https://github.com/spine-generic/data-multi-subject/pull/176 and https://github.com/spine-generic/data-multi-subject/pull/177), so we just need to make sure that your scripts work after these changes. If so, we could either point to a specific commit, as you suggested. Or maybe create a new release of the spine-generic dataset and point to this release?

Reply:
> Or maybe create a new release of the spine-generic dataset and point to this release?

I feel like this would be a better solution! @sandrinebedard what do you think? Could you make a new release summarizing the changes to the `spine-generic` dataset?

Reply:
I plan to create a new release of spine generic this week!

Reply:
perfect, thank you! please tag me when you do so that I can add the link to my script

Reply:
Here is the release! https://github.com/spine-generic/data-multi-subject/releases/tag/r20250310

Reply:
done here! https://github.com/sct-pipeline/contrast-agnostic-softseg-spinalcord/pull/139/commits/200466350a1ce6c134b00fdc9d151e768a2acc8e